data/scenarios/Challenges/ice-cream.yaml

```diff
@@ -0,0 +1,242 @@
+version: 1
+name: Ice Cream Shop
+author: Karl Ostmo
+description: |
+  Serve a demanding customer
+creative: false
+seed: 0
+objectives:
+  - goal:
+      - |
+        Congratulations on the grand opening of your new ice cream shop.
+        You have advertised: "All you can eat, for 1 bitcoin!"
+      - |
+        A customer is approaching the "Counter". They look hungry!
+      - "..."
+      - |
+        Customer: "`give` me a cone, and then I'll tell you how many scoops I want."
+      - "..."
+      - |
+        Oh dear, you've forgotten to stock your shop with a "calculator".
+        Let's hope this order is simple.
+    condition: |
+      customer <- robotnamed "customer";
+      as customer {has "cone"};
+  - goal:
+      - |
+        `give` the customer exactly what they request.
+      - |
+        Perhaps there is something else around the shop that can assist with
+        preparing the order...
+    condition: |
+      as base {
+        has "bitcoin";
+      };
+robots:
+  - name: base
+    dir: [0, 1]
+    devices:
+      - branch predictor
+      - treads
+      - antenna
+      - comparator
+      - workbench
+      - grabber
+      - dictionary
+      - lodestone
+      - lambda
+      - logger
+      - welder
+      - hearing aid
+      - net
+      - strange loop
```

Comment:
Do we really need all this? In case of an error, the inventory would scroll to the logger, so it's hard to see the acquired items. That may have been another reason why I missed the briefcase. :sweat_smile:

Reply:
The only two that are not essential are `lodestone` and `net`.  I've removed those, but added a `scanner` per @byorgey's [comment](https://github.com/swarm-game/swarm/pull/1008#discussion_r1070611027).